tools/mavparmdiff.py

```diff
@@ -9,13 +9,23 @@
 parser = ArgumentParser(description=__doc__)
 parser.add_argument("file1", metavar="FILE1")
 parser.add_argument("file2", metavar="FILE2")
+parser.add_argument("--use-excludes",
+                    help="exclude volatile and similar parameters",
+                    default=True,
+                    action='store_true',
+                    dest='use_excludes')
+parser.add_argument("--no-excludes",
+                    help="include volatile and similar parameters",
+                    default=True,
```

Comment:
I think I've read that putting default in both options isn't recommended when the destination is the same.

Reply:
I agree